tests/test_tools.py

```diff
@@ -623,7 +653,8 @@ def test_env_path():
 
 def test_env_path_slices():
     # build os-dependent paths properly
-    mkpath = lambda *paths: os.sep + os.sep.join(paths)
+    def mkpath(*paths):
+        return os.sep + os.sep.join(paths)
```

Comment:
I can happily revert, but PEP8 complained¹.
If this is a general thing, it might make sense to put this into the Developer's Guide.

---

<sup>¹ "Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier." [(source with reasons)](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)</sup>